saleor/static/js/components/cart.js

```diff
@@ -57,6 +57,19 @@ export default $(document).ready((e) => {
       }
     });
   });
+  $('.cart__clear').click((e) => {
+    $.ajax({
+      url: $('.cart__clear').attr('action'),
+      method: 'POST',
+      data: {},
+      success: (response) => {
+        $('.badge').html(response.numItems)
+        $.cookie('alert', 'true', {path: '/cart'});
+        location.reload();
+        deliveryAjax();
```

Comment:
Is it really necessary? In addition, it is called after instructing the browser to reload the page.